cogs/tas2gif.py

```diff
@@ -0,0 +1,131 @@
+from discord.ext import commands
+from discord.ext import tasks
+import discord
+import requests
+from config import celia_home
```

Comment:
The code assumes that celia_home ends with a "/", but paths aren't usually specified like that in configuration (though maybe discord.py is different, I wouldn't know). This should either be communicated clearly or preferably the code should assume that it doesn't end in a "/". After all `/directory//file` usually results in the same thing as `/directory/file` but `/directoryfile` will definitely result in an error.